datasets/switchboard.py

```diff
@@ -160,24 +160,89 @@ def run(self):
                     )  # speaker_id gender recording
 
 
+class CreateLDCSwitchboardSpeakerListJob(Job):
+    """
+    This creates the speaker list according to the conversation and speaker table
+    from the LDC documentation: https://catalog.ldc.upenn.edu/docs/LDC97S62
+
+    The resulting file contains 520 speakers in the format of:
+        <speaker_id> <gender> <recording>
+    """
+
+    def __init__(self, caller_tab_file, conv_tab_file):
```

Comment:
pls add docstring as to what files should be input here (if it is not immediately obvious)